glean-core/src/metrics/string_list.rs

```diff
@@ -133,4 +134,17 @@ impl StringListMetric {
         let value = Metric::StringList(value);
         glean.storage().record(&self.meta, &value);
     }
+
+    /// **Test-only API (exported for FFI purposes).**
+    ///
+    /// Get the currently-stored values as a JSON String.
```

Comment:
nit: what's the expected format? `["string1", "string2", ..., "string20"]`? If so, can you please add it here, in the docs, as an example?